.env

```diff
@@ -24,14 +24,14 @@ POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
 
 # InfluxDB
-INFLUXDB_HOST=localhost
+INFLUXDB_HOST=host.docker.internal
```

Comment:
`host.docker.internal` one only works on mac or windows, not on linux

See also mongo uri down there.

Reply:
seems like this one can be override on linux by adding:
```yaml
  extra_hosts:
    - host.docker.internal:host-gateway
```
on the services where you need it.

Reply:
note: that it is a bit tricky though for the service you want to reach, needs to listen on the internal address corresponding to your docker sub network (this may be done by listening all interface, but even though it should listen live on newly created interfaces if your network is destroyed / recreated, like it happens on a down / up)